src/main/java/sandbox/SqlSandboxUtils.java

```diff
@@ -0,0 +1,61 @@
+package sandbox;
+
+import org.sqlite.javax.SQLiteConnectionPoolDataSource;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.sql.SQLException;
+
+/**
+ * Methods that mostly are used around the mocks that replicate stuff from
+ * other projects.
+ *
+ * @author ctsims
+ * @author wspride
+ */
+public class SqlSandboxUtils {
+
+    public static void deleteDatabaseFolder(String path) {
+        File databaseFolder = new File(path);
+        if (databaseFolder.exists()) {
+            deleteFolder(databaseFolder);
+        }
+    }
+
+    public static boolean databaseFolderExists(String path) {
+        File file = new File(path);
+        return file.exists();
+    }
+
+    public static void deleteFolder(File folder) {
+        File[] files = folder.listFiles();
+        if (files != null) { //some JVMs return null for empty dirs
+            for (File f : files) {
+                if (f.isDirectory()) {
+                    deleteFolder(f);
+                } else {
+                    f.delete();
+                }
+            }
+        }
+        folder.delete();
+    }
+
+    public static SQLiteConnectionPoolDataSource getDataSource(String databaseName, String databasePath) {
+        File databaseFolder = new File(databasePath);
+
+        try {
+            if (!databaseFolder.exists()) {
+                Files.createDirectories(databaseFolder.toPath());
+            }
+            Class.forName("org.sqlite.JDBC");
+            SQLiteConnectionPoolDataSource dataSource = new SQLiteConnectionPoolDataSource();
+            dataSource.setUrl("jdbc:sqlite:" + databasePath + "/" + databaseName + ".db");
```

Comment:
https://github.com/dimagi/formplayer/blob/d78489612008305bf520e046220c180f52db7227/src/main/java/services/FormplayerStorageFactory.java#L65-L65
https://github.com/dimagi/formplayer/blob/d78489612008305bf520e046220c180f52db7227/src/main/java/services/RestoreFactory.java#L161-L161

These are the only two places they come from so I feel p. good